lib/bundler.js

```diff
@@ -91,11 +91,19 @@ function bundleInstructions(instructions, bundler) {
    b.require(instructionsStream, { expose: 'loopback-boot#instructions' });
    */
 
+  var instructionId = 'instructions';
+  // Create an unique instruction identifier using the application ID.
+  // This is only useful when multiple loopback applications are being bundled
+  // together.
+  if (instructions.applicationId)
```

Comment:
I see, you are converting `appId` to `applicationId` below. Ignore the first part of the comment then.